docs/source/contribution.rst

```diff
@@ -341,7 +341,15 @@ How to Write Tests
 ~~~~~~~~~~~~~~~~~~
 
 There are many examples of unit tests under the :tree:`tests` directory, so reading some of them is a good and recommended way to learn how to write tests for Chainer.
-They simply use the ``unittest`` package of the standard library, while some tests are using utilities from :mod:`chainer.testing`.
+They simply use the :mod:`unittest` package of the standard library, while some tests are using utilities from :mod:`chainer.testing`.
+
+In addition to the :ref:`coding-guide` mentioned above, the following rules are applied to the test code:
+
+* All test classes must inherit from :class:`unittest.TestCase`.
+* In addition to the :mod:`unittest` features, you can also use the following ``pytest`` features.
```

Comment:
"should" or "must" instead of "can"?

Reply:
I don't think so. New tests should be written in the new convention. We will simply loosen the review criteria.
Also it will take too long (perhaps never) to rewrite all the existing tests.

Reply:
OK. I think `should` can be used along with the following excuse. Do you think it's OK?

```
We're incrementally applying the above style.
Some existing test codes may using old style (`self.assertRaises`, etc.), but all newly written test code should follow the above style.
```

Reply:
Yes, I think it's good to include that excuse.

Some minor corrections:
* "test codes" -> "tests"
* "may using" -> "may be using"
* "old style" -> "the old style"
* "newly written test code" -> "newly written tests"

Reply:
Thanks! I updated accordingly.